packages/gui/playwright-report/index.html

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
This file was deleted in main. Do we need it? I was thinking we should just add it to .gitignore if it keeps coming back.